### PR TITLE
gitmoji: update `desc` and `homepage`

### PR DESCRIPTION
**Pull request by carloscuesta**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Hello! 👋🏼 

This PR updates the homepage of the `gitmoji-cli` project that changed last year and also the description of the package to be more precise with it

Related with https://github.com/carloscuesta/gitmoji/issues/645

### CODE REVIEW

File: Formula/gitmoji.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 require "language/node"
 
 class Gitmoji < Formula
-  desc "Emoji guide for your commit messages"
-  homepage "https://gitmoji.carloscuesta.me"
+  desc "Interactive command-line tool for using gitmoji on commits"
```
Comment (nandahkrishna) [inline Formula/gitmoji.rb:4]: Is this intentionally `gitmoji` and not `emoji`? The repository description is also the old description so I'm unsure why we want to change it (seems clear as-is).

**Review by nandahkrishna — COMMENTED** [review]
Thanks for your PR. I've squashed your commits together because we do not allow merge commits in homebrew-core. Just one comment/clarification.

File: Formula/gitmoji.rb (hunk 1 above)
Comment (carloscuesta) [reply Formula/gitmoji.rb:4]: The description that is right now, refers to the gitmoji repository.

This formula is not an emoji guide, is a command line tool for using gitmoji on git commits that’s why I considered that the description should be updated 🙂

Btw, I’m not sure who added this first description (not me) but in my opinion is not accurate to what the gitmoji-cli tool is doing

Comment (carloscuesta) [reply Formula/gitmoji.rb:4]: I wanted to use the same description that I’m using in: https://github.com/carloscuesta/gitmoji-cli

But it was not possible as descriptions can’t start with an article or the formula name so I had to tweak it a little bit

Comment (nandahkrishna) [reply Formula/gitmoji.rb:4]: I think we can use:
```suggestion
  desc "Interactive command-line tool for using emoji in commit messages"
```
(Apologies for the delayed response.)

**Review by nandahkrishna — APPROVED** [review]
Thanks!

**Pull request merged**